style.css

```diff
@@ -43,14 +37,14 @@ body {
   display: flex;
   flex-wrap: wrap;
   justify-content: space-evenly;
-  width: 15rem;
+  width: 16rem;
 }
 .nav-link {
   font-size: 19px;
   text-decoration: none;
   border-radius: 10px;
   color: var(--font-primary);
```

Comment:
The contrast ratio is pretty bad at 2.9 with this color on the blue nav background.